package com.luciano.cadastropessoa.cadastrarpessoa.exception

class AuthorNotFoundException(id: Long) : RuntimeException("N√ÉO EXISTE ESSE ID TESTE ERRO: $id")
